recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -180,12 +197,11 @@ def upload_package(feedstock_root, recipe_root, config_file, validate, private,
     specific_config = safe_load(open(config_file))
     if "channel_targets" in specific_config:
         channels = [c.strip().split(" ") for c in specific_config["channel_targets"]]
-        source_channels = ",".join(
-            [c.strip() for c in specific_config["channel_sources"]])
+        source_channels = _get_channel_names(specific_config["channel_sources"])
```

Comment:
I don't understand this change.

Reply:
Yes, i misunderstood. This does not need to be adapted. I removed the changes to the upload section